recipe/meta.yaml

```
@@ -16,21 +15,20 @@ source:
   # but without losing metadata
   # Please note when using conda-build locally, may get following error: certifi cannot depend on itself
   # Try pushing and refer to Prefect logs
-  - url: https://pypi.io/packages/py3/p/pip/pip-{{ pip_version }}-py3-none-any.whl
+  - url: https://pypi.org/packages/py3/p/pip/pip-{{ pip_version }}-py3-none-any.whl
     sha256: 3790624780082365f47549d032f3770eeb2b1e8bd1f7b2e02dace1afa361b4ed
     folder: pip_wheel
-  - url: https://pypi.io/packages/py3/s/setuptools/setuptools-{{ setuptools_version }}-py3-none-any.whl
+  - url: https://pypi.org/packages/py3/s/setuptools/setuptools-{{ setuptools_version }}-py3-none-any.whl
     sha256: ce74b49e8f7110f9bf04883b730f4765b774ef3ef28f722cce7c273d253aaf7d
     folder: setuptools_wheel
 
 build:
   number: 0
-  skip: True  # [py<37]
+  skip: true  # [py<37]
 
 requirements:
   host:
     - python
```

https://github.com/certifi/python-certifi/blob/2025.11.12/pyproject.toml#L2
```suggestion
    - python
    - wheel
    - setuptools >=42.0.0
```

---

Not in this case. See: https://github.com/AnacondaRecipes/certifi-feedstock/blob/master/recipe/meta.yaml#L15

---

ok, something new for me, I need to investigate how it works from the inside.